store/dataProviders.js

```diff
@@ -0,0 +1,30 @@
+import { action, observable } from 'mobx'
+
+class DataProviders {
+  @observable selectedProvider = {
+    id: 1,
+    name: 'Open Research Online',
+  }
+
+  allProviders = new Map([])
+
+  constructor(rootStore) {
+    this.rootStore = rootStore
+  }
+
+  @action
+  retrieveDataProviders = async term => {
+    // TODO: Invalidate cache after some time
+    //       Move to @oacore/api
+    if (this.allProviders.get(term)) return this.allProviders.get(term)
+    const r = await fetch(
+      `https://api.core.ac.uk/internal/organisations?q=${term}`
+    )
```

Comment:
It should be something like `https://api.core.ac.uk/internal/users/{id}/data-providers`

Reply:
Yep this was done long time ago. We need to change it but sadly it's out of scope for this PR.